nabs/utils.py

```diff
@@ -110,3 +116,108 @@ def add_named_kwargs_to_signature(
     )
 
     return sig.replace(parameters=start_params + wrapper_params + end_params)
+
+
+def format_ophyds_to_html(obj, allow_child=False):
```

Comment:
allow_child isn't documented below.

API thought - you might consider just accepting `*objs` instead of sequence-or-instance.

Reply:
whoops thanks